lib/forest_liana/engine.rb

```diff
@@ -16,8 +16,17 @@ def configure_forest_cors
       begin
         rack_cors_class = Rack::Cors
         rack_cors_class = 'Rack::Cors' if Rails::VERSION::MAJOR < 5
+        null_regex = Regexp.new(/\Anull\z/)
 
         config.middleware.insert_before 0, rack_cors_class do
+          allow do
+            hostnames = [null_regex, 'localhost:4200', /\A.*\.forestadmin\.com\z/]
+            hostnames += ENV['CORS_ORIGINS'].split(',') if ENV['CORS_ORIGINS']
+
+            origins hostnames
+            resource ForestLiana::AuthenticationController::PUBLIC_ROUTES[1], headers: :any, methods: :any, credentials: true, max_age: 86400 # NOTICE: 1 day
+          end
```

Comment:
[MISC] This looks like duplicated with the following block, is that intended? If so, I might clarifications :)

Reply:
Yes it is intended, we add a rule to the cors for a specific ressource (`null_regex`), it's because when redirected from a browser the origin is not sent, so we allow `null` origin for a specific route.